### DISCUSSION TITLE
Ability to choose/set animation style as either "ping-pong" or loop in server settings.

### DISCUSSION THREAD

**Discussion by PyroTech03** [AscensionGameDev/Intersect-Engine#2084] [Ideas]
I'm working with premade assets that are easy for me to work with in a generator.  By default the gen I'm using uses the Ping-Pong frame style, 1-2-3-2-1-2..., While Intersect uses a 4 frame loop, 1-2-3-4-1-2-3-4.

If we could get a toggle to allow us to choose loop or ping-pong in the server config.  

This would allow us to use a sprite sheet such as this one from the Time Elements set.

![light_full_body_walking](https://github.com/AscensionGameDev/Intersect-Engine/assets/13249558/c1066b7e-466c-4b4c-93f3-4ecaa4eb94bf)

**Comment by Ezekyuhl** [comment]
Copy the middle one and add it to the start, set frames to 4.

**Reply by PyroTech03** [reply]
I'm already doing this. Was thinking about ways to reduce work in an art program.

Many assets on sale are in the 3 frame format so thought it'd be a good option.
